fdbrpc/FlowTransport.actor.cpp

```diff
@@ -988,6 +988,7 @@ ACTOR static Future<Void> connectionIncoming( TransportData* self, Reference<ICo
 ACTOR static Future<Void> listen( TransportData* self, NetworkAddress listenAddr ) {
 	state ActorCollectionNoErrors incoming;  // Actors monitoring incoming connections that haven't yet been associated with a peer
 	state Reference<IListener> listener = INetworkConnections::net()->listen( listenAddr );
+	state int64_t connectionCount = 0;
```

Comment:
```suggestion
	state uint64_t connectionCount = 0;
```

Overflow on unsigned is defined, but overflow on int64 is undefined behavior.  I know it's an int64, but like, might as well not open that door?